sormas-backend/src/main/java/de/symeda/sormas/backend/event/EventGroupFacadeEjb.java

```diff
@@ -432,15 +440,14 @@ public void linkEventToGroup(Event event, User currentUser, List<EventGroup> eve
 				"User " + currentUser.getUuid() + " is not allowed to link events from another region to an event group.");
 		}
 
-		// Check that the event group is not already related to this event
-		List<EventGroup> filteredEventGroups = getFilteredEventGroups(event, eventGroups);
+		List<EventGroup> eventGroupsToLinkTo = getEventGroupsNotLinkedTo(event, eventGroups);
```

Comment:
Related to the last acceptance criteria, I'm not sure if the linking should not be possible if the user has rights to link the event but the jurisdiction is different but the user is the owner of the event ( since is crossed the "owned by them" part).
I've checked the code and tested it, and right now is possible, as it was in the past.
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/dd5e4d4d-ced8-4585-af36-36dc338eb319)

I've changed the jurisdiction of the above used Test User from Nation to Region level (Hamburg):
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/18b49093-df26-4f97-ae73-c4d0135e34d7)
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/f1eb2d40-d1bb-490f-9d5c-5ce0dae7ea4e)
I was able to link an event with the same disease as the user has (the owner is the logged in test user) from another jurisdiction ( if the owner part is still requested, everything is fine).
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/764d7d8d-3ee2-4464-99ee-f24b11ac5c28)

![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/1e590936-91f1-4d71-87f7-d6f2cddd1b0f)

Also tried to link another event ( with a different owner and different region but same disease).
The save is prevented with an Access Denied exception as it is checked in the code:
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/49a7c651-fa0f-444a-88d3-50482c7d611d)

What is not clear, why this event appears at all in the Event section ( different jurisdiction ,different owner, same disease) but the display of this event in Event Directory is not related to this ticket:
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/d75d8d10-49f8-4cd3-88a9-93629c6d2097)

![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/9516aa40-f501-48d7-b5cc-f869d5e632e8)
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/020525b9-9129-4dfb-95da-6069cbbf1aaf)